crates/extension_cli/Cargo.toml

```diff
@@ -16,7 +16,7 @@ path = "src/main.rs"
 anyhow.workspace = true
 clap = { workspace = true, features = ["derive"] }
 env_logger.workspace = true
-extension = { workspace = true, features = ["no-webrtc"] }
```

Comment:
Why did we remove the `no-webrtc` feature?

Is there some other mechanism that is preventing the extension CLI from depending on WebRTC now?

Reply:
Yeah, extension doesn’t depend on workspace anymore!